notebooks/231-instruct-pix2pix-image-editing/231-instruct-pix2pix-image-editing.ipynb

```diff
@@ -0,0 +1,1062 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #37.        vae_decoder = mo.convert_model()
</pre><p>vae_decoder = mo.convert_model(VAE_DECODER_ONNX_PATH)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/796/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
thank you, fixed